### ISSUE TITLE
Screen reader is not announcing the name for the 'progress bar' control in Windows.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#2184]
**Describe the bug**
Screen reader is not announcing the name for the 'progress bar' control in Windows. Screen reader users will not get the proper context of the control to interactive control if it doesn't announce name.

**To Reproduce**

1. Open the URL: [Blazor ProgressBar Component | Free UI Components by Radzen](https://blazor.radzen.com/progressbar?theme=material#progressbar-determinate) in latest edge browser.
2. Progress bar page will appear.
3. Press tab key to reach to the progress bar.
4. Verify on pressing arrow key, whether screen reader is announcing name for the 'Progress bar' control or not.

**Expected behavior**
On pressing arrow key, 'Progress' control receives focus screen reader should announce the name with 'Progress bar 55' in Windows.

**Screenshots**

https://github.com/user-attachments/assets/ae55ebf3-e899-424e-8e01-2f496d8edc7a

<img width="976" alt="Image" src="https://github.com/user-attachments/assets/b046bfbb-9356-45e8-aca5-7c0734a6622e" />

**Desktop (please complete the following information):**
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Edge Version 130.0.2849.27

**Additional context**
NA
